lib/elastic/app-search/client/engines.rb

```diff
@@ -21,6 +22,12 @@ def create_engine(engine_name, language = nil)
         def destroy_engine(engine_name)
           delete("engines/#{engine_name}")
         end
+
+        def update_documents(engine_name, documents)
```

Comment:
I'm not sure this method should be in engines, it looks like it's a separate API https://swiftype.com/documentation/app-search/api/documents

Reply:
Looks like a freeloader 😠